device-types/Dell/VxRail-P670F.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Dell
+model: VxRail P670F
+slug: dell-vxrail-p670f
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+weight: 35.2
+weight_unit: kg
+front_image: true
+rear_image: true
+comments: |
+  [Dell VxRail installation](https://dl.dell.com/content/manual44265077-dell-vxrail-p670f-p670n-v670f-and-s670-installation-and-service-manual.pdf?language=en-us)
+  [Dell VxRail datasheet](https://dl.dell.com/content/manual43304433-dell-vxrail-p670f-p670n-v670f-and-s670-technical-specifications.pdf?language=en-us)
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
```

Comment:
These appear to be modular. Is that true? If so then please model them in `module-bays` and ensure the associated PSUs are modelled in the module-types folder